app/controllers/supervisor_volunteers_controller.rb

```diff
@@ -19,7 +19,7 @@ def unassign
     volunteer = Volunteer.find(params[:id])
     supervisor_volunteer = volunteer.supervisor_volunteer
     supervisor_volunteer.is_active = false
-    supervisor_volunteer.save
+    supervisor_volunteer.save!
```

Comment:
Yay! We might want to add more error handling later but this is much better than failing silently- like it was doing before.